golang/pkg/btcspv/bitcoin_spv.go

```diff
@@ -347,28 +341,28 @@ func ExtractMerkleRootBE(header []byte) []byte {
 }
 
 // ExtractTarget returns the target from a given block hedaer
-func ExtractTarget(header []byte) sdk.Int {
+func ExtractTarget(header []byte) sdk.Uint {
 	// nBits encoding. 3 byte mantissa, 1 byte exponent
 	m := header[72:75]
 	e := sdk.NewInt(int64(header[75]))
```

Comment:
should be `NewUint`?

Reply:
Actually, if I remember right, there was one place where it needed to be int

Reply:
Oh, got it. It's because of the weird integer wrapping thing they do